src/templates/Header/Header.module.scss

```diff
@@ -0,0 +1,8 @@
+.graylinks {
+    align-items: center;
+    display: flex;
+    font-family: 'Roboto Mono', "Helvetica","Arial", sans-serif;
+    font-size: 16px;
+    gap: 16px; 
+    letter-spacing: 0.15px;
```

Comment:
let's use rem instead of px

Reply:
Resolved in 97545bb
Didn't use rem for letter spacing as it would most likely involve a loss of precision